tutorials/colors/colorbar_only.py

```diff
@@ -0,0 +1,112 @@
+"""
+=============================
+Customized Colorbars Tutorial
+=============================
+
+This tutorial shows how to build colorbars without an attached mappable.
+
+"""
+
+###############################################################################
+# Customized Colorbars
+# ====================
+#
+# `matplotlib.colorbar.ColorbarBase` derives from `ScalarMappable` and puts a
+# colorbar in specified axes, it is the base class with standalone colorbar
+# drawing functionality. It can be used as-is to make a colorbar for a given
+# colormap and does not need a mappable object like an image. In this tutorial
+# we will explore what can be done with standalone colorbar.
+#
+# We will start by making a figure of desired size and adding axis at position
+# [left, bottom, width, height] where all quantities are in fractions of figure
+# width and height.
+
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+fig = plt.figure(figsize=(8, 3))
```

Comment:
I wonder if you can get away with this since axes stuff isn't the point of this tutorial:
`fig, (ax1, ax2, ax3) = plt.subplots(nrows=3, figsize=(8,3))`